### ISSUE TITLE
Tab counting is not working

### ISSUE THREAD

**Issue by DavidLD-GitHub** [42School/norminette#324]
Norminette does not count the tabs properly after an if statement 

**Additional infos**
 - OS: macOs Catalina
 - python --version: 2.7.16
 - norminette -v: 3.3.51

**Comment by Jkutkut** [comment]
I've been doing my own testing and I believe I know what's going on. The problem appears to be that any assignment done inside a scope other than the function's can have any indentation. Therefore, the amount of tabs can be chosen freely.

In the following example, the code complies with flags, runs and is norminette@v3.3.51 valid (keep in mind that the tab size in GitHub is different than in 42).

```
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   control_indentation_bug.c                          :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: jre-gonz <jre-gonz@student.42madrid>       +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2022/08/28 10:11:47 by jre-gonz          #+#    #+#             */
/*   Updated: 2022/08/28 11:31:07 by jre-gonz         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

void	ft_if(void)
{
	int	var;

	if (1 == 1)
															var = 12312;
	if (1 == 1)
	{
	var = 123;
			var = 123;
																	var = 123;
					var = 123;
		var = 123;
							var = 123;
												var = 123;
	}
	var = 1;
	while (var--)
		if (1 - 1)
											var = var + 1 * 0;
	var = 1;
	while (var--)
		if (1 - 1)
	var = var + 1 * 0;
}

void	ft_else(void)
{
	int	var;

	if (0 == 0)
		var = 1;
	else
	{
	var = 123;
																	var = 123;
					var = 123;
		var = 123;
												var = 123;
	}
	var = 0;
	while (var--)
		var = var - 1 * 0;
	var = 0;
	while (var--)
	{
	var = var - 1 * 0;
								var = var - 1 * 0;
		var = var - 1 * 0;
												var = var - 1 * 0;
	}
}

void	ft_random(void)
{
	int	i;

	{
	i = 0;
			i = 0;
							i = 0;
					i = 0;
										i = 0;
		{
									i = 0;
							i = 0;
											i = 0;
					i = 0;
										i = 0;
		}
	}
}

int	main(void)
{
	ft_if();
	ft_else();
	ft_random();
	return (0);
}
```

```
$> gcc -Wall -Wextra -Werror control_indentation_bug.c -o bug && ./bug
$> norminette
control_indentation_bug.c: OK!
```

This assignment can call a function. However, if the function is called by itself, the invalid indentation is detected.

```
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   control_indentation_bug_2.c                        :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: jre-gonz <jre-gonz@student.42madrid>       +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2022/08/28 11:38:41 by jre-gonz          #+#    #+#             */
/*   Updated: 2022/08/28 11:42:15 by jre-gonz         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

int	ft_one(void)
{
	return (1);
}

void	ft_void(void)
{
}

void	ft_random(void)
{
	int	i;

	{
		i = ft_one();
			i = ft_one();
			ft_void();
	}
}

int	main(void)
{
	ft_random();
	return (0);
}
```

```
$> gcc -Wall -Wextra -Werror control_indentation_bug.c && ./a.out
$> norminette
control_indentation_bug_2.c: Error!
Error: TOO_MANY_TAB         (line:  29, col:   1):	Extra tabs for indent level
```

**Note**: Line 29 is the line where `ft_void()` is called.

## Additional infos

- Mac OS (42Madrid):
  - norminette 3.3.51
  - python 3.7.3
- Ubuntu 21.10:
  - norminette 3.3.51 and repository in current [commit](https://github.com/42School/norminette/commit/d443df72accb7b66bd65e98d76207da1e6968e00).
  - python 3.9.2
- Debian GNU/Linux 11.
  - norminette 3.3.51
  - python 3.9.2

**Comment by matthieu42Network** [comment]
fixed in 3.3.52
